client/packages/reports/src/ListView/ListView.tsx

```diff
@@ -63,6 +70,16 @@ export const ListView = () => {
           reportWithArgs={reportWithArgs}
           setReportWithArgs={setReportWithArgs}
         />
+        {store?.preferences?.omProgramModule && (
```

Comment:
maybe also check that there are some reports? Otherwise there's a blank widget shown 🤷

Reply:
I've added a condition on all widgets to hide if no reports 😉

Reply:
Also if you wanna test it with Stock status reports~  https://github.com/msupply-foundation/open-msupply-reports/pull/71